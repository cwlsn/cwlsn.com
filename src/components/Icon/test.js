import React from 'react'
import renderer from 'react-test-renderer'
import Icon from '.'

test('Icon', () => {
	const tree = renderer
		.create(<Icon icon="github" />)
		.toJSON()

	expect(tree)
		.toMatchSnapshot()
})

test('Icon renders without death', () => {
	const tree = renderer
		.create(<Icon icon="github" />)
		.root

	expect(tree.findAllByType(Icon)).toHaveLength(1)
})

