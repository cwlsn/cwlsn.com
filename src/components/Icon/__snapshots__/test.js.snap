// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon 1`] = `
<test-file-stub
  icon="github"
/>
`;
